api/src/orgs/data.js

```diff
@@ -30,10 +30,35 @@ export function getOrganisations (
     )
   }
 
+  const query = db.collection('orgs')
   return makeConnection(
-    db.collection('orgs'),
+    query,
     args,
     filter,
     camelToSnakeCaseKeys(args.sort)
   )
+    .then(augmentWithOrgStats(query, { ...filter }))
+}
+
+const augmentWithOrgStats = (query, filter) => async function (obj) {
+  return {
+    ...obj,
+    ...(await makeOrgStats(query, filter))
+  }
+}
+
+async function makeOrgStats (query, filter) {
```

Comment:
```suggestion
async function fetchOrgStats (query, filter) {
```

I think this is a more descriptive name